src/pages/NotesArchives/FilledNotesArchives.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./NotesArchives.css";
+
+const FilledNotesArchives = ({displayList}) => {
+    return (
+        <>
+            <h4 className='text-center fw-700'>ARCHIVED NOTES</h4>
+            <div className="gentle-grid-responsive py-8 m-4">
+                {/* {
```

Comment:
remove this commented code if not needed!!

Reply:
I don't know why this snippet of code appears to be commented when I have uncommented it in my local